pkg/cloud/alibaba/assets/cloud-controller-manager-clusterrole.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cloud-controller-manager
```

Comment:
We already have a cloud-controller-manager role don't we? You'll need to give this a different name and create a clusterrolebinding as well

Reply:
we do, it didn't occur to me there was a name collision here.  I'll work on this.